tests/view_tests/tests/test_debug.py

```diff
@@ -557,6 +559,21 @@ def __str__(self):
         text = reporter.get_traceback_text()
         self.assertIn('USER: [unable to retrieve the current user]', text)
 
+    @override_settings(ROOT_URLCONF='view_tests.urls')
```

Comment:
nope, not needed, I was using the decorator to make a 404 error as in the above comment and then did not remove it